build/yaml/crd/nsx.vmware.com_ipaddressallocations.yaml

```diff
@@ -3,7 +3,8 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.14.0
+    controller-gen.kubebuilder.io/version: v0.11.0
```

Comment:
maybe rebase the code? I think this is not related to your patch

Reply:
it is strange this is downgraded

Reply:
we should not downgrade it, should revert it.

Reply:
The controller-gen version in the main branch is v0.11.0. I wondering why the version here is v0.14.0
```
CONTROLLER_GEN = $(shell pwd)/bin/controller-gen
.PHONY: controller-gen
controller-gen: ## Download controller-gen locally if necessary.
	$(call go-get-tool,$(CONTROLLER_GEN),sigs.k8s.io/controller-tools/cmd/controller-gen@v0.11.0)
```
https://github.com/vmware-tanzu/nsx-operator/blob/4bc8285df1e1cb4dac2a0255488fa15686022ae1/Makefile#L149

Reply:
if main branch is v.0.11.0, we can revert it.